src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -745,6 +746,10 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		}
 	}
 
+	private async onTableKeyDown(event: ITableKeyboardEvent) {
+		this.table.setActiveCell(event.cell.row, event.cell.cell);
```

Comment:
This is going to fire anytime any key is pressed - is that intentional?

Reply:
Alphabetical keys didn't do anything, and keys like `Ctrl` + `C` still copied successfully when a cell in result grid was in focus.

Would you recommend that I only be listening for `End`/`Home` key presses?

Reply:
You should definitely only be reacting to keys that you intend to have do something. Very easy for bugs to slip in otherwise (and also generally just unnecessary overhead)

Reply:
Do you know how we map keyCodes? Fox example, in the `ITableKeyboardEvent` that is being caught. The key code for the `End` key is 35, and in the `KeyCode` enum found in KeyCodes.ts, the key code for `End` is 13. I'm not sure how the value is converted, but I'll scan around for that.